src/components/jscommon/CanvasWidget/CanvasWidgetNew.tsx

```diff
@@ -127,26 +132,21 @@ const CanvasWidget = <T extends BaseLayerProps>(props: Props<T>) => {
 
     const _handleMouseMove = useCallback((e: React.MouseEvent<HTMLCanvasElement, MouseEvent>) => {
         const { point, mouseButton, modifiers } = ClickEventFromMouseEvent(e, ClickEventType.Move)
-        dispatch({type: COMPUTE_DRAG, mouseButton: mouseButton === 1, point: point, shift: modifiers.shift || false})
-        _dispatchDragEvents(false)
+        dispatchDrag({type: COMPUTE_DRAG, mouseButton: mouseButton === 1, point: point, shift: modifiers.shift || false})
         _dispatchDiscreteMouseEvents(e, ClickEventType.Move)
-    }, [_dispatchDragEvents, _dispatchDiscreteMouseEvents])
+    }, [_dispatchDiscreteMouseEvents])
```

Comment:
Does the drag region still animate correctly after removing the `_dispatchDragEvents()` call here? (Like the draglayer in `ElectrodeGeometryTest`?) Or does the `useEffect()` hook pick it up?